integration-cli/docker_cli_help_test.go

```diff
@@ -279,87 +245,59 @@ func (s *DockerSuite) TestHelpExitCodesHelpOutput(c *check.C) {
-	}
+	c.Assert(out, checker.Not(checker.HasSuffix), "\n\n", check.Commentf("Should not have a blank line at the end of 'docker'\n"))
 
 	// docker help: stdout=all, stderr=empty, rc=0
-	cmd = exec.Command(dockerBinary, "help")
-	stdout, stderr, ec, err = runCommandWithStdoutStderr(cmd)
-	if len(stdout) == 0 || len(stderr) != 0 || ec != 0 || err != nil {
-		c.Fatalf("Bad results from 'docker help'\nec:%d\nstdout:%s\nstderr:%s\nerr:%v", ec, stdout, stderr, err)
-	}
+	out, _, err = dockerCmdWithError("help")
+	c.Assert(err, checker.IsNil, check.Commentf(out))
 	// Be really pick
-	if strings.HasSuffix(stdout, "\n\n") {
-		c.Fatalf("Should not have a blank line at the end of 'docker help'\n%s", stdout)
-	}
+	c.Assert(out, checker.Not(checker.HasSuffix), "\n\n", check.Commentf("Should not have a blank line at the end of 'docker help'\n"))
 
 	// docker --help: stdout=all, stderr=empty, rc=0
-	cmd = exec.Command(dockerBinary, "--help")
-	stdout, stderr, ec, err = runCommandWithStdoutStderr(cmd)
-	if len(stdout) == 0 || len(stderr) != 0 || ec != 0 || err != nil {
-		c.Fatalf("Bad results from 'docker --help'\nec:%d\nstdout:%s\nstderr:%s\nerr:%v", ec, stdout, stderr, err)
-	}
+	out, _, err = dockerCmdWithError("--help")
+	c.Assert(err, checker.IsNil, check.Commentf(out))
 	// Be really pick
-	if strings.HasSuffix(stdout, "\n\n") {
-		c.Fatalf("Should not have a blank line at the end of 'docker --help'\n%s", stdout)
-	}
+	c.Assert(out, checker.Not(checker.HasSuffix), "\n\n", check.Commentf("Should not have a blank line at the end of 'docker --help'\n"))
 
 	// docker inspect busybox: stdout=all, stderr=empty, rc=0
 	// Just making sure stderr is empty on valid cmd
-	cmd = exec.Command(dockerBinary, "inspect", "busybox")
-	stdout, stderr, ec, err = runCommandWithStdoutStderr(cmd)
-	if len(stdout) == 0 || len(stderr) != 0 || ec != 0 || err != nil {
-		c.Fatalf("Bad results from 'docker inspect busybox'\nec:%d\nstdout:%s\nstderr:%s\nerr:%v", ec, stdout, stderr, err)
-	}
+	out, _, err = dockerCmdWithError("inspect", "busybox")
+	c.Assert(err, checker.IsNil, check.Commentf(out))
 	// Be really pick
-	if strings.HasSuffix(stdout, "\n\n") {
-		c.Fatalf("Should not have a blank line at the end of 'docker inspect busyBox'\n%s", stdout)
-	}
+	c.Assert(out, checker.Not(checker.HasSuffix), "\n\n", check.Commentf("Should not have a blank line at the end of 'docker inspect busyBox'\n"))
 
 	// docker rm: stdout=empty, stderr=all, rc!=0
 	// testing the min arg error msg
-	cmd = exec.Command(dockerBinary, "rm")
-	stdout, stderr, ec, err = runCommandWithStdoutStderr(cmd)
-	if len(stdout) != 0 || len(stderr) == 0 || ec == 0 || err == nil {
-		c.Fatalf("Bad results from 'docker rm'\nec:%d\nstdout:%s\nstderr:%s\nerr:%v", ec, stdout, stderr, err)
-	}
+	cmd := exec.Command(dockerBinary, "rm")
+	stdout, stderr, _, err := runCommandWithStdoutStderr(cmd)
+	c.Assert(err, checker.NotNil)
+	c.Assert(len(stderr), checker.Not(checker.Equals), 0)
```

Comment:
I'm *meh* on this one as we're checking the content below.. At least we should add a comment but i think it could be removed.